views/index.blade.php

```diff
@@ -28,14 +28,37 @@
 
 <div id="graphiql">Loading...</div>
 <script src="{{ \MLL\GraphiQL\DownloadAssetsCommand::jsPath() }}"></script>
+<script src="{{ \MLL\GraphiQL\DownloadAssetsCommand::explorerPluginPath() }}"></script>
 <script>
+    var fetcher = GraphiQL.createFetcher({
```

Comment:
```suggestion
    const fetcher = GraphiQL.createFetcher({
```

I doubt any developer will use a browser not supporting const